src/playlist.js

```diff
@@ -374,30 +375,54 @@ export const seekable = function(playlist, expired, liveEdgePadding) {
  * @param {number} startTime
  * @return {Object}
  */
-export const getMediaInfoForTime = function(
+export const getMediaInfoForTime = function({
```

Comment:
suggestion: the jsdoc for this method should be updated as well to reflect the new options object.
```
* @param {Object} options.playlist the media playlist to query
* @param {number} options.currentTime The number of seconds since the earliest
* possible position to determine the containing segment for
* @param {number} options.startIndex
* @param {number} options.startTime
* @param {number} options.segmentIndex
* @param {number} options.partIndex
* ```

Reply:
I guess right now `partIndex should be `{?number}`

Reply:
I think it's `{number} [partIndex]` to denote optional.

Reply:
yeah, `[partIndex]` to be option, but given that it can be `null` or a number, it should have the `?`.